TombEngine/Game/gui.cpp

```diff
@@ -2381,6 +2398,23 @@ namespace TEN::Gui
 				lara->Weapons[(int)LaraWeaponType::Crossbow].SelectedAmmo = WeaponAmmoType::Ammo3;
 		}
 
+		if (lara->Weapons[(int)LaraWeaponType::HK].Present)
+		{
+			lara->Weapons[(int)LaraWeaponType::HK].Weaponmode = LaraWeaponTypeCarried::WTYPE_AMMO_1;
+			lara->Weapons[(int)LaraWeaponType::HK].SelectedAmmo = WeaponAmmoType::Ammo1;
+
+			if (Ammo.CurrentHKAmmoType == 1)
+			{
+				lara->Weapons[(int)LaraWeaponType::HK].Weaponmode = LaraWeaponTypeCarried::WTYPE_AMMO_2;
+				lara->Weapons[(int)LaraWeaponType::HK].SelectedAmmo = WeaponAmmoType::Ammo1;
```

Comment:
Is this intended that `SelectedAmmo` is also `Ammo1` here, or it is copypaste error?